front/src/app/navigation/map-view/map-view.component.ts

```diff
@@ -135,6 +135,8 @@ export class MapViewComponent implements AfterViewInit {
       }
     }
 
+    mapHeightNeeded += 100;
```

Comment:
why? :smile:

Reply:
:D geb @070108111 recht.. Wieso?

und wenn der Grund valide is => Kommentar 👯‍♀️

Reply:
sonst kann man nicht ganz nach unten scrollen wenn der bottom drawer im bottom state ist :joy: ich schau mal ob ich noch eine bessere Lösung finde ;)